cpp-data-aggregation/aggregation_consumer.cpp

```diff
@@ -123,32 +121,32 @@ void run_aggregation_consumer(std::ofstream& timing_file,
 
 int main(int argc, char* argv[]) {
 
-    std::string context("Scaling tests");
-
     MPI_Init(&argc, &argv);
 
     //initializing rank
     int rank;
     MPI_Comm_rank(MPI_COMM_WORLD, &rank);
 
-    log_data(context, LLDebug, "Starting Data Aggregation test");
+    std::string context("Data Aggregation Tests Consumer Rank " + std::to_string(rank));
+    log_data(context, LLDebug, "Rank initialized");
+    log_data(context, LLDebug, "Starting Data Aggregation tests");
     double main_start = MPI_Wtime();
 
     // Get command line arguments
-    if(argc==1)
-        // log_error(context, LLInfo, "The number tensor size in "\
-        //                          "bytes must be provided as "\
-        //                          "a command line argument.");
+    if(argc==1) {
+        log_error(context, LLInfo, "The number tensor size in " \
+                                 "bytes must be provided as " \
+                                 "a command line argument.");
```

Comment:
Same here actually. This will have the side effect of improving the usefulness of the exception